### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by ghost**
Tests for products_aggregator.

### CODE REVIEW

File: apps/main/admin.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def queryset(self, request, queryset):
             return queryset.filter(size__icontains=self.value())
 
 
-class US_Sizes(SimpleListFilter):
+class USSizes(SimpleListFilter):
```
Comment (sibiryoff) [inline apps/main/admin.py]: ??? Why it is changed within this task?

File: apps/main/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        product_with_image = Product.objects.get(id=2)
+        self.assertEquals(product_without_image.image_exists(), False)
+        self.assertEquals(product_with_image.image_exists(), True)
+
+
+class UploadedFileModelTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        mixer.blend(UploadedFile, file='uploaded_files/WOMEN_SHOES.csv')
```
Comment (sibiryoff) [inline apps/main/tests/test_models.py]: Do not place test file in MEDIA_ROOT folder.

File: apps/main/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        def is_active(self):
+            return True
+
+        def is_staff(self):
+            return True
+
+    request_factory = RequestFactory()
+    request = request_factory.get('/admin')
+    request.user = MockSuperUser()
+    request.files = [settings.MEDIA_ROOT + 'uploaded_files/WOMEN_SHOES.csv',
```
Comment (sibiryoff) [inline apps/main/tests/test_views.py]: Same as in previous comment.

File: apps/main/tests/test_views.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            content_type=ct
+        )
+        group_supervisers.permissions.add(permission_for_supervisers)
+        test_supervisor = User.objects.create_user(username='Supervisor', password='12345678')
+        test_supervisor.groups.add(group_supervisers)
+
+    def test_add_upload_file_for_supervisers(self):
+        self.client.login(username='Supervisor', password='12345678')
+        resp = self.client.get('/admin/main/uploadedfile/add/')
+        self.assertEquals(resp.status_code, 302)
```
Comment (sibiryoff) [inline apps/main/tests/test_views.py:117]: What about access for non-Superviser users?

**Comment by sibiryoff** [comment]
Great work. Let's focus on comments before final approval.

**Comment by ghost** [comment]
On this moment tests failed, because pull request bugfix/08 don't merge into master.

**Comment by sibiryoff** [comment]
@leocoan please make sure that all tests can be run by `python manage.py test` command.

**Comment by ghost** [comment]
@sibiryoff now command `python manage.py test` run all test in the all application. Now we have one application - apps.main.

**Review by sibiryoff — APPROVED** [review]

**Pull request merged**